1.11/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -89,6 +93,45 @@ This topic provides all available configuration parameters. Except where explici
 | [zk_master_credentials](#zk-master)          | ([enterprise type="inline" size="small" /]) The ZooKeeper master credentials.  |
 | [zk_agent_credentials](#zk-agent)           | ([enterprise type="inline" size="small" /]) The ZooKeeper agent credentials.  |
 
+### adminrouter_tls_1_0_enabled
+
+Indicates whether to enable TLS 1.0 on Admin Router on master nodes. Changing this setting has no effect on internal Admin Router configuration on agent nodes.
+
+- `adminrouter_tls_1_0_enabled: 'true'` Enable the TLS 1.0 protocol on Admin Router.
+- `adminrouter_tls_1_0_enabled: 'false'` Disable the TLS 1.0 protocol on Admin Router. This is the default value.
+
+You are advised not to enable TLS 1.0 as the protocol is considered insecure.
+
+If you have already installed your cluster and would like to change this in-place, you can go through an [upgrade][3] with the `adminrouter_tls_1_0_enabled` parameter set to the desired value.
+
+### adminrouter_tls_1_1_enabled
+
+Indicates whether to enable TLS 1.1 on Admin Router on master nodes. Changing this setting has no effect on internal Admin Router configuration on agent nodes.
+
+- `adminrouter_tls_1_1_enabled: 'true'` Enable the TLS 1.1 protocol on Admin Router master node. This is the default value.
+- `adminrouter_tls_1_1_enabled: 'false'` Disable the TLS 1.1 protocol on Admin Router master node.
+
+If you have already installed your cluster and would like to change this in-place, you can go through an [upgrade][3] with the `adminrouter_tls_1_1_enabled` parameter set to the desired value.
+
+### adminrouter_tls_1_2_enabled
+
+Indicates whether to enable TLS 1.2 on Admin Router on master nodes. Changing this setting has no effect on internal Admin Router configuration on agent nodes.
+
+- `adminrouter_tls_1_2_enabled: 'true'` Enable the TLS 1.2 protocol on Admin Router master node. This is the default value.
+- `adminrouter_tls_1_2_enabled: 'false'` Disable the TLS 1.2 protocol on Admin Router master node.
+
+It is advised to keep this protocol version enabled as its most secure widely supported TLS version.
+
+If you have already installed your cluster and would like to change this in-place, you can go through an [upgrade][3] with the `adminrouter_tls_1_2_enabled` parameter set to the desired value.
+
+### adminrouter_tls_cipher_suite
+
+Provide a custom list of TLS cipher suites. The value will be passed directly into Admin Router's [`ssl_ciphers`](http://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_ciphers) configuration directive. There is no validation of this string. Setting it incorrectly will cause DC/OS installation to fail. This configuration settings affects only Admin Router running on DC/OS master nodes.
```

Comment:
Is it clear enough do you think what happens if this is not set?

Reply:
"only Admin Router running on DC/OS master nodes" - I think that what you have here is right but want to be sure, is it "Admin Routers"?

Reply:
Added following sentence:

```
If not provided the default value `EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5;` is used.
```